File: Java/Employee Salary Calculation/Employee.java
public class Employee{
    private int employeeId;
    private String employeeName;
    private double salary;
    private double netSalary;
    public void setEmployeeId(int id){
        this.employeeId=id;
    }
    public int getEmployeeId(){
        return employeeId;
    }
    public void setEmployeeName(String name){
        this.employeeName=name;
    }
    public String getEmployeeName(){
        return employeeName;
    }
    public void setSalary(double sal){
        this.salary=sal;
    }
    public double getSalary(){
        return salary;
    }
    public void setNetSalary(double net){
        this.netSalary=net;
    }
    public void calculateNetSalary(int pfp){
        double netSalary=salary-(salary*pfp/100);
        setNetSalary(netSalary);
    }
    public double getNetSalary(){
        return netSalary;
    }
}

File: Python/Employee Salary/employee.py
class Employee:
    def __init__(self,employee_id,employee_name,basic_pay):
        self.employee_id=employee_id
        self.employee_name=employee_name
        self.basic_pay=basic_pay
    def calculate_net_salary(self):
        da=self.basic_pay*10/100
        ta=self.basic_pay*5/100
        hra=self.basic_pay*8/100
        pf=self.basic_pay*10/100
        net_sal=(self.basic_pay+da+ta+hra)-pf
        return net_sal

eid=input("Enter the employee id:")
ename=input("Enter the employee name:")
basic_pay=float(input("Enter the basic pay:"))
emp_obj=Employee(eid,ename,basic_pay)
print("\nEmployee Id:",emp_obj.employee_id)
print("Employee Name:",emp_obj.employee_name)
net_salary=emp_obj.calculate_net_salary()
print("Net Salary :%.2f" %net_salary)